Analysis/Tasks/PWGUD/upcForward.cxx

```diff
@@ -0,0 +1,111 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/*
+I ran this code using following:
+o2-analysis-timestamp| o2-analysis-upc-forward  --aod-file <path to ao2d.txt> [--isPbPb]  --aod-memory-rate-limit 10000000000 --shm-segment-size 16000000000 -b
+for now AO2D.root I am using is
+/alice/data/2015/LHC15o/000246392/pass5_lowIR/PWGZZ/Run3_Conversion/138_20210129-0800_child_1/0001/AO2D.root
+it can be copied using
+alien_cp alien:/alice/data/2015/LHC15o/000246392/pass5_lowIR/PWGZZ/Run3_Conversion/138_20210129-0800_child_1/0001/AO2D.root  file:./
+*/
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "AnalysisDataModel/TrackSelectionTables.h"
+#include "AnalysisDataModel/EventSelection.h"
+#include <TH1D.h>
+#include <TH2D.h>
+#include "TLorentzVector.h"
+#include "AnalysisCore/TriggerAliases.h"
+using namespace std;
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+#define mmuon 0.1057 //mass of muon
+struct UPCForward {
+  //defining histograms
+  OutputObj<TH1D> hMass{TH1D(
```

Comment:
Is there any particular reason for not having the histogram declaration on a single line?

Reply:
It was formatted like this when I ran clang-format otherwise there is no particular reason for that.

Reply:
I changed the format now.